.github/workflows/push.yaml

```diff
@@ -0,0 +1,84 @@
+name: OpenFisca-Core / Pull request review
+
+on:
+  pull_request:
+    types: [assigned, opened, reopened, synchronize, ready_for_review]
```

Comment:
After this PR, the CI will not run on push outside a Pull Request.

Curent behavior is to run on all push : `on: [ push, pull_request, workflow_dispatch ]`.

Reply:
Yes, I removed it to easy review. But I can readd it. I'll do.